x-pack/legacy/plugins/lens/public/indexpattern_plugin/dimension_panel/field_select.tsx

```diff
@@ -140,6 +140,7 @@ export function FieldSelect({
     <EuiComboBox
       fullWidth
       compressed
+      isClearable={false}
```

Comment:
The clear icon inside this combo box has the same behavior as the X button to the right of the label- while I can see how it's confusing to have a button that looks like "close", maybe the solution is to put an explicit "Clear column" button inside the popover?

Reply:
It doesn't **quite** act as the same thing as the clearing of the entire configuration. All it clears is the field selected but it doesn't change the selected aggregation. I also don't think it's absolutely necessary to to clear the whole selection once inside the popover. There's really only 2 selections to make (aside from adjusting the aggregation settings). The main problem I'm solving was users trying to close the popover using this x, which would then clear their field selection and without an undo, could be very frustrating.